### ISSUE TITLE
Formula calculation exception

### ISSUE THREAD

**Issue by tristanleboss** [PHPOffice/PhpSpreadsheet#1391]
This is:

```
- [X] a bug report
- [ ] a feature request
- [X] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?

I expect to get value "1"

### What is the current behavior?

An exception is raised.

### What are the steps to reproduce?

![image](https://user-images.githubusercontent.com/1431508/75100341-903fa180-55cc-11ea-9dc5-38a039425dc2.png)

Use provided Excel file : [Simplebug.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/4240216/Simplebug.xlsx)

```php
<?php

require_once('./vendor/autoload.php');

$reader = new \PhpOffice\PhpSpreadsheet\Reader\Xlsx();
$spreadsheet = $reader->load("Simplebug.xlsx");

echo 'A1: '.$spreadsheet->getActiveSheet()->getCell('A1')->getCalculatedValue().PHP_EOL;
die;
```

### Which versions of PhpSpreadsheet and PHP are affected?

Last version installed by Composer.

**Comment by tristanleboss** [comment]
Still an issue... should not be closed.

**Comment by tristanleboss** [comment]
Automatic activity ;)

**Comment by phannphong** [comment]
I have same problem. Anyone know how to fix this? Or any suggestion please?

**Comment by tristanleboss** [comment]
MarkBaker assigned the issue to itself so it's probably because he plans to work on it at some point.

I kept the issue open because I like to see issues in a project before using it... if the stale bot closes them, you have the illusion the project is error free when it's not.

It's probably a problem with either the accented letter (é, eacute) or the simple quote (')... what's the name of the problematic sheet?

**Comment by tristanleboss** [comment]
Removing the simple quote in the sheet name solve the issue so the problem is with sheet names with simple quote in it ;)

**Comment by tristanleboss** [comment]
You're an annoying bot ;)

**Comment by MarkBaker** [comment]
Should have been resolved in latest master branch by [PR #2651](https://github.com/PHPOffice/PhpSpreadsheet/pull/2651)
